pkg/goDB/DBWorkManager.go

```diff
@@ -237,23 +277,21 @@ func (w *DBWorkManager) readBlocksAndEvaluate(workload DBWorkload, resultMap has
 		var (
 			blocks      [types.ColIdxCount][]byte
 			blockBroken bool
-			ts          int64
-			iface       string
 		)
 
 		// Read the blocks from their files
 		for _, colIdx := range query.columnIndices {
 
 			// Read the block from the file
-			if blocks[colIdx], err = dir.ReadBlockAtIndex(colIdx, b); err != nil {
+			if blocks[colIdx], err = workDir.ReadBlockAtIndex(colIdx, b); err != nil {
 				blockBroken = true
-				w.logger.Warnf("[D %s; B %d] Failed to read column %s: %s", dir, block.Timestamp, types.ColumnFileNames[colIdx], err.Error())
+				w.logger.Warnf("[D %s; B %d] Failed to read column %s: %s", workDir, block.Timestamp, types.ColumnFileNames[colIdx], err.Error())
```

Comment:
Refresh my memory: these should actually be logged to `stderr` with the logging overhaul of `goQuery`, right?

Reply:
I guess, yeah, up to a certain point / number (if possible)...